assets/un_street_map.yml

```diff
@@ -0,0 +1,518 @@
+version: 8
+name: UN Street Map
+metadata: {}
+sprite:
+  - id: undp
+    url: ./sprite/sprite
+  - id: default
+    url: https://pro-ags2.dfs.un.org/arcgis/rest/services/Hosted/Unite_StreetMapVT_CVW_V01/VectorTileServer/resources/sprites/sprite_S
+glyphs: https://fonts.undpgeohub.org/fonts/{fontstack}/{range}.pbf
+sources:
+  esri:
+    type: vector
+    attribution: '&copy; UN Street Map (UN)'
```

Comment:
change attribution to `&copy; <a href="https://www.un.org/geospatial/mapsgeo" target="_blank" rel="noopener">United Nations</a>`